app/pages/verificationwithpassword/verificationwithpassword.js

```diff
@@ -81,11 +88,15 @@ export let VerificationWithPassword = React.createClass({
   },
 
   render: function() {
+    if (_.get(this.props, 'notification.message', '') === errorMessages.ERR_BIRTHDAY_MISMATCH) {
+      this.props.trackMetric('VCA Home Verification - Birthday Mismatch')
+    }
```

Comment:
Side effects in a `render` method make me 😧. Definitely more appropriate in `componentWillReceiveProps` at worst, at best is there a way to fold it into the redux actions going on?